doc/tutorials/acrn_on_qemu.rst

```diff
@@ -217,61 +217,26 @@ Install ACRN Hypervisor
 Bring-Up User VM (L2 Guest)
 ***************************
 
-1. Build the ACRN User VM kernel.
+1. Build the User VM disk image (``UOS.img``) following :ref:`build-the-ubuntu-kvm-image` and copy it to the ACRNSOS (L1 Guest).
```

Comment:
Can we take this opportunity to also rename `UOS.img` into `UserVM.img`?

Reply:
Done！